x/ibc/light-clients/07-tendermint/types/tendermint.pb.go

```diff
@@ -5,6 +5,11 @@ package types
 
```

Comment:
Why was this file re-generated?

Reply:
see https://github.com/cosmos/cosmos-sdk/issues/7562.